### ISSUE TITLE
"Comment" when approving reservations

### ISSUE THREAD

**Issue by Daniel-AVE** [TIHLDE/kontres#89]
Would like to have the possibility of adding a comment when a reservation is approved in order to let user know of important information regarding their reservation. An example of this would be someone who's reserved the office during a weekday, it would be nice to be able to add a comment saying "Remember that your reservation is in a weekday, and according to the guidelines for using the office, the office will have to be cleaned before xx:xx tomorrow". 

There are definitely other use-cases where being able to add a comment would be relevant.

**Comment by MindChirp** [comment]
Skal ta for meg dette i frontend nå. @ConradOsvik kan du mekke backenden for denne? Tror vi må diskutere litt hvordan vi skal gå fram for å løse det. Funker det å snakkes en gang før neste møte?

**Comment by ConradOsvik** [comment]
Har fri fra torsdag til søndag og er down for et møte. Skal sette meg inn i backend men vet ikke om jeg får fikset før møtet.

**Comment by MindChirp** [comment]
@Daniel-AVE @ConradOsvik Denne er ferdig i frontend nå, venter på backend

**Comment by MindChirp** [comment]
@Daniel-AVE Går det greit om beskjeden du skriver er synlig for alle som går inn på reservasjonen? 
![image](https://github.com/user-attachments/assets/ec56e9ae-60f7-4f1d-9415-9108ca5f41b0)

**Comment by Daniel-AVE** [comment]
Ja, burde gå greit det. Om det ikke er en tanke å inkludere kommentaren i varselet / e-posten som sendes når en godkjenner/avslår reservasjoner? Om det går?
